flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/mailbox/TaskMailboxProcessorTest.java

```diff
@@ -117,6 +119,8 @@ public void runDefaultAction(Controller controller) throws Exception {
         MailboxProcessor mailboxProcessor = start(mailboxThread);
         mailboxProcessor.getMailboxExecutor(DEFAULT_PRIORITY).execute(() -> stop.set(true), "stop");
         mailboxThread.join();
+        Assert.assertTrue(counter.get() > 0);
+        Assert.assertTrue(mailboxProcessor.getNumMailsProcessedCounter().getCount() > 0);
```

Comment:
`assertGE(mailboxProcessor.getNumMailsProcessedCounter().getCount(), counter.get())`?

Reply:
counter.get() is not correct here as counter is the number of default action executions, not the number of mails. At the moment, the number of mails processed includes only the poison mail and manual the stop mail. So it would be 2. However, if there are changes in future that some house-keeping mails are generated and processed, one would need to adjust this test assertion. The > 0 proves that the counter is working in general. I can change it to > 1 which is IMO here better than =2.